CHANGELOG.adoc

```diff
@@ -7,34 +7,31 @@ All notable changes to this project are documented in this file.
 
 The format is based on {uri-changelog}[Keep a Changelog].
 
-= Unreleased
+= v3.0.0 (September 21,2021)
 
 == New features
-* Changed default tag values
-* Renamed operator_tags --> freeform_tags
-* Renamed notification variables
-* Renamed variable operating_system_version --> operator_os_version
-* Renamed variable operator_upgrade --> upgrade_operator
-* Renamed variable timezone --> operator_timezone
-* Renamed variable operator_instance_principal --> enable_operator_instance_principal
-* Added support for Bastion service
-* AD lookup mechanism reimplemented to remove dependency on deprecated template_file data source
-* Replaced deprecated template_file data source with templatefile function
+* Added support for ARM Flex Shape (#30)
```

Comment:
This should be "Ampere A1 Flex shape" or at least "Arm" as per their trademark usage guidelines.